### DISCUSSION TITLE
fprettify warning output is written inside the script

### DISCUSSION THREAD

**Discussion by Koushikphy** [krvajal/vscode-fortran-support#508] [Q&A - deprecated see Fortran Discourse] [answered]
I posted this issue https://github.com/pseewald/fprettify/issues/123 in `fprettify` repo. Don't know if it's the problem with the `fprettify` or this extension itself.

**Comment by gnikit** [comment]
I suspect this might have something to do with how the output of stderr is handled, instead of being thrown in the log channel it gets passed on stdout and written on your doc.

**Comment by Koushikphy** [comment]
When I use `fprettify` directly from terminal, it writes the error on the terminal/stderr and does not modify the script itself. So may be the problem is how this extension uses `fprettify` ?

**Reply by gnikit** [reply]
Yes, that is what I meant above. The two outputs might not be properly split (in the vscode ext) hence the mixup.

**Comment by gnikit** [comment, marked as answer]
This has now been fixed, see https://github.com/fortran-lang/vscode-fortran-support/issues/517 and it will be in the upcomming v3.0 release.
I am marking this discussion as answered feel free to comment if you have more questions

**Reply by Koushikphy** [reply]
Thanks for the quick fix
